AnnaAaBrekke/portofolio
# Set up Development Tools (Vite, ESLint, Prettier, Husky, lint-staged)

- [x] Vite: Added as the dev server and build tool.
- [x] ESLint: Configured for linting JavaScript files.
- [x] Prettier: Set up for automatic code formatting with custom rules (double quotes, trailing commas, and semicolons).
- [x] Husky: Configured with a pre-commit hook to run linting and formatting checks.
- [x] lint-staged: Ensures ESLint and Prettier only run on staged files for efficiency.

Testing:

1. Run npm run dev to test Vite.
2. Use npm run lint to check for linting issues.
3. Use npm run format to format the code.
4. Test the pre-commit hook by staging and committing files to see ESLint and Prettier in action.
